### DISCUSSION TITLE
我的小记 - 知识沉淀

### DISCUSSION THREAD

**Discussion by giscus** [xiaodongxier/wiki#6] [Announcements]
# 我的小记 - 知识沉淀

王永杰的个人网站,王永杰的个人wiki,小东西儿的个人网站,xiaodongxier.com,wangyongjie.cn,前端知识管理,个人wiki,个人知识管理

http://127.0.0.1:8000/%20%E9%A6%96%E9%A1%B5/%E6%88%91%E7%9A%84%E5%B0%8F%E8%AE%B0/

**Comment by xiaodongxier** [comment]
DaedalOS 跑在浏览器里的桌面环境 #开源 #浏览器

用 JavaScript 写的运行在浏览器中的仿 Windows 桌面操作系统网站，支持开始菜单、动态壁纸、命令行终端、视频播放器、3D 弹球、Markdown 查看器、浏览器等功能。

https://dustinbrett.com/ https://github.com/DustinBrett/daedalOS

**Comment by xiaodongxier** [comment]
- BpmnElement - 一种图表
- DAG - 有向无环图（DAG）

**Comment by xiaodongxier** [comment]
首页文件夹路径及文件名不能修改， 影响评论的显示 😅

**Comment by xiaodongxier** [comment]
Mac重启访达 ：
方法1: ` killall Finder `
方法2: ` option+cmmand+ESC `

**Comment by xiaodongxier** [comment]
MacBook USB 连 iPhone 响个不停，充电频繁的连上/断开解决方案：  ` sudo killall -STOP -c usbd `

**Comment by xiaodongxier** [comment]
https://github.com/aoaostar/toolbox
⚙️傲星工具箱 - 一款开源的在线工具箱程序

▶️演示地址 | 📖小白教程

📄经常用到在线工具箱，是否想过自己部署一个呢？傲星工具箱可以满足你的想法

内置众多小工具，带后台管理系统，并支持插件管理，体验不错

**Comment by xiaodongxier** [comment]
https://www.videofk.com/zh-CN3252

**Reply by xiaodongxier** [reply]
VideoFk - 一个在线视频下载工具

📄支持海内外 36 个平台的视频下载，免费且速度可观

**Comment by xiaodongxier** [comment]
https://github.com/imsyy/home

个人主页，我的个人主页，个人主页源码，主页模板，homepage

![](https://gitcdn.xiaodongxier.com/wiki/20240221233818.png)

**Comment by xiaodongxier** [comment]
百度 jquery cdn
```
https://libs.baidu.com/jquery/1.9.0/jquery.js
```

**Comment by xiaodongxier** [comment]
1、less和less-loader版本问题
可以直接按照以下命令安装即可（一定要严格控制版本号，以下版本可以完美解决依赖冲突问题）：
npm install less@3.9.0 less-loader@4.1.0 --save-dev或
cnpm install less@3.9.0 less-loader@4.1.0 --save-dev（这个是针对配置了淘宝镜像的安装方法）

作者：coderSlow
链接：https://juejin.cn/post/7043601099034460168
来源：稀土掘金
著作权归作者所有。商业转载请联系作者获得授权，非商业转载请注明出处。
